package.json

```diff
@@ -15,13 +15,14 @@
   "main": "platform/node/index.js",
   "repository": {
     "type": "git",
-    "url": "git://github.com/maplibre/maplibre-gl-native.git"
+    "url": "git+https://github.com/maplibre/maplibre-gl-native.git"
   },
   "license": "BSD-2-Clause",
   "dependencies": {
     "@mapbox/cmake-node-module": "^1.2.0",
+    "@mapbox/node-pre-gyp": "^1.0.9",
+    "@acalcutt/node-pre-gyp-github": "1.4.8",
     "minimatch": "^3.0.4",
-    "node-pre-gyp": "^0.10.2",
```

Comment:
node-pre-gyp was recently bumped from 0.10 -> 0.17, which makes a conflict. It should of course have been replaced by "@mapbox/node-pre-gyp". I'll make a PR to move main to @mapbox/node-pre-gyp so it becomes easier to resolve for this branch.

Reply:
I remove node-pre-gyp in favor of @mapbox/node-pre-gyp already. This is a removed line right or am I missing something?

Reply:
You're not missing something. The node-pre-gyp was already upgraded to 0.17 in main, making a conflict, but main is on @mapbox/node-pre-gyp now to relieve it. This branch doesn't have to contain that specific change anymore.